packages/integration-tests/projects/suite-web/support/utils/shortcuts.ts

```diff
@@ -110,3 +110,16 @@ export const enableRegtestAndGetCoins = ({ payments = [] }) => {
     });
     cy.task('mineBlocks', { block_amount: 1 });
 };
+
+export const createAccount = (coin: string, accountNameAdd: string) => {
```

Comment:
just an idea. I would maybe be using longer and more descriptive names for shortcuts in form <do something - from somewhore>
so here it would be `createAccountFromDashboard` ? 
this is just an idea for discussion. feel free to ignore this

Reply:
Renamed.